Choose PDF From List/ChoosePDFFromList.csproj

```diff
@@ -65,7 +65,19 @@
 	</ItemGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="Syncfusion.Maui.PdfViewer" Version="21.1.37" />
+		<PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiVersion)" />
+		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
+		<PackageReference Include="Syncfusion.Compression.NET" Version="*" />
```

Comment:
It is not required to include all the syncfusion PDF Viewer packages when you refer from **Nuget org**. Please check and optimize

Reply:
Optimised